ocs_ci/ocs/cluster.py

```diff
@@ -743,3 +744,41 @@ def validate_osd_utilization(osd_used=80):
             logger.warn(f"{osd} used value {value}")
 
     return _rc
+
+
+def get_node_resource_utilization(nodename=None):
+    """
+    Gets the node's cpu and memory utilization in percentage
+
+    Args:
+        nodename (str) : The node name
+
+    Returns:
+        dict : Node name and its cpu and memory utilization in
+               percentage
+
+    """
+
+    node_names = [nodename] if nodename else [
+        node.name for node in get_typed_nodes(node_type='worker')
```

Comment:
Add param for node_type, can use this function to get utilization of `master` nodes too, default can be `worker`